@xen-orchestra/proxy/config.toml

```diff
@@ -21,7 +21,12 @@ keepAliveInterval = 10e3
 dirMode = 0o700
 snapshotNameLabelTpl = '[XO Backup {job.name}] {vm.name_label}'
 
-[backups.defaultSettings]
+[backups.defaultSettings.metadata]
+reportWhen = 'failure'
+retentionPoolMetadata = 0
+retentionXoMetadata = 0
+
+[backups.defaultSettings.vm]
```

Comment:
I suggest we keep `backup.defaultSettings` for common settings and `backups.metadata.defaultSettings` and `backups.vm.defaultSettings` for specific settings.

What do you think?